notification/src/handler/notification/notification.handler.js

```diff
@@ -253,13 +256,17 @@ function compareTransactionStates(currentState, newState) {
 }
 
 function getAddInterfaceInteractionUpdateAction(notification) {
-  // strip away sensitive data
-  delete notification.additionalData
-  delete notification.reason
+  const moduleConfig = config.getModuleConfig()
+  const notificationToUse = _.cloneDeep(notification)
+  if (!moduleConfig.logSensitiveData) {
+    // strip away sensitive data
+    delete notificationToUse.NotificationRequestItem.additionalData
+    delete notificationToUse.NotificationRequestItem.reason
```

Comment:
here we are removing `reason` but do not ensuring it in test cases, also I'm not really sure if the reason is sensitive data, it might something related to the refusals. Would be better to ask Adyen imo. I see some things also in this doc, search with `reason` : https://docs.adyen.com/development-resources/webhooks/understand-notifications

Reply:
This code is old, I guess there is a reason that `reason` is considered as sensitive data. I added verification for it: https://github.com/commercetools/commercetools-adyen-integration/pull/745/commits/d75b0510387a74d4890f4c44327b71666b38cca6